### ISSUE TITLE
[navSide] no :hover state on navSide-item-link with Safari < 15.4

### ISSUE THREAD

**Issue by BertrandPodevin** [LuccaSA/lucca-front#1620]
The apparence of the `:hover` state of `.navSide-item-link` is broken on older version of Safari

This is due to the présence of the unsupported `:focus-visible` in the same CSS rule

![image](https://user-images.githubusercontent.com/65592565/164487434-ea6eb65b-1d0a-40b3-aa6b-f4511a5e1e1b.png)

![image](https://user-images.githubusercontent.com/65592565/164488291-80ad191a-ddcd-4087-9dee-e387b753bf79.png)

**Comment by BertrandPodevin** [comment]
Can be fixed by just separating the 2 rules

```
.navSide-item-link {
   &:focus-visible {...}
   &:hover{...}
}
```

**Comment by jeremie-lucca** [comment]
Fixed (8.1.5) https://github.com/LuccaSA/lucca-front/pull/1628
